biz.aQute.bnd.gradle/src/aQute/bnd/gradle/Resolve.groovy

```diff
@@ -136,7 +137,16 @@ public class Resolve extends DefaultTask {
         throw new GradleException("${run.getPropertiesFile()} workspace errors")
       }
 
+      def parent = run.parent;
       try {
+        if (parent instanceof Processor) {
+          Properties gradleProperties = new PropertiesWrapper()
+          gradleProperties.put('task', this)
+          gradleProperties.put('project', project)
+          Processor processor = new ProcessorWrapper(gradleProperties)
```

Comment:
Why do you need ProcessorWrapper? Isn't `new Processor(gradleProperties, false)` enough?